src/sentry/tasks/reports.py

```diff
@@ -393,21 +393,53 @@ def build_message(timestamp, duration, organization, user, report):
     return message
 
 
+DISABLED_ORGANIZATIONS_USER_OPTION_KEY = 'reports:disabled-organizations'
+
+
+def user_subscribed_to_organization_reports(user, organization):
+    return organization.id not in set(
+        UserOption.objects.get_value(
+            user=user,
+            project=None,
+            key=DISABLED_ORGANIZATIONS_USER_OPTION_KEY,
+            default=[],
+        )
+    )
+
+
+class Skipped(object):
```

Comment:
This only exists for testing purposes and I felt an identity check was safer (although I suppose anybody could just come by and change these values to other types anyway and lose the benefit.)